main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.highbit",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_highbit,		php_core_globals, 		core_globals)
```

Comment:
> so what is the other security aspect besides escape sequences? On the other hand, not allowing UTF-8 to pass nowadays is weird.

@weltling Regarding, `syslog.ascii_only` anyone who wants to can set this to `false`.